dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenTracing-2.0/io.openliberty.mpOpenTracing-2.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpOpenTracing-2.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-ShortName: mpOpenTracing-2.0
+Subsystem-Name: MicroProfile OpenTracing 2.0
+IBM-API-Package: \
+    org.eclipse.microprofile.opentracing; type="stable"
+-features=\
+    io.openliberty.opentracing-2.0, \
+    io.openliberty.org.eclipse.microprofile.opentracing-2.0, \
+    com.ibm.websphere.appserver.mpConfig-1.4
```

Comment:
MP 4.0 features should only depend on other MP 4.0 features. In this case, please change the dependency to MP Config 2.0.

Reply:
MicroProfile OpenTracing 2.0-RC1 is still using Config 1.4.  Will change this when MicroProfile OpenTracing updates its dependency. See https://github.com/eclipse/microprofile-opentracing/blob/9b79905dd7946cef8a404387b80d99db71e36ebd/pom.xml#L36

Reply:
ok